src/test/java/com/github/mauricioaniche/ck/util/JDTUtilsTest.java

```diff
@@ -0,0 +1,44 @@
+package com.github.mauricioaniche.ck.util;
+
+import junit.framework.Assert;
+import org.junit.jupiter.api.Test;
+import java.util.Arrays;
+
+import static com.github.mauricioaniche.ck.util.JDTUtils.getVariableName;
+
+public class JDTUtilsTest {
+    @Test
```

Comment:
Indeed a lot of effort to make it work for all the different ways of declaring a variable

Reply:
This test is redundant with the new implementation of getVariableName.